src/note/models.py

```diff
@@ -21,15 +21,31 @@ def get_queryset(self):
         )
 
 
-class Note (models.Model):
+class NotePriority(models.Model):
+    DEFAULT_PRIORITY_ID = 1
```

Comment:
@aaboffill  My primary point is: when I read `DEFAULT_PRIORITY_ID = 1`, it is harder for me to find what it means than using, for example, a function to lazy-evaluate the default value.

Sometimes it's possible that PK doesn't start from 1, or even PK is a UUID field or a string field... but in our context it's safe. We may need to add a restriction on Django admin, to disable the "delete" button for the default values.

I have no preference on the both implementations. @manumilou Do you have a preference?